### DISCUSSION TITLE
Review VPD Roles and make it better

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4881] [General]
We need to think about separating roles by collection vs. across all collections.

**Comment by Jegelewicz** [comment]
See https://github.com/ArctosDB/arctos/issues/3659

**Comment by Jegelewicz** [comment]
Ability to view transactions, but not edit also.

**Comment by mkoo** [comment]
This impacts a lot of usability issues and with move to PG we may have more, or at least different, options available to manage VPD functions. Due to PG dependency, adding to SABI award project
